_includes/head.html

```diff
@@ -134,5 +134,15 @@
     }
   </script>
 
-  <link rel="stylesheet" href="{{ '/css/main.css' | relative_url }}">
+  <meta name="theme-color" content="#111111">
```

Comment:
```suggestion
  <meta name="theme-color" content="#ae00ee">
```

Using the primary color for theme as opposed to black